guidelines/dxp/metal-plugins/metal-aop.md

```diff
@@ -0,0 +1,70 @@
+# Metal AOP guide of use
```

Comment:
I'm not sure how to document this, but we should avoid `metal` references... 

My original idea was something like `/patterns/dxp/aspect_oriented_programming.md` or if `AOP` is not a clear enough reference, something named after the use cases...

Reply:
@wincent, @julien, @bryceosterhaus, input on this?